src/helpers.ts

```diff
@@ -48,8 +48,12 @@ export const symbols = {
   failed: red('✖'),
 };
 
+export function generateHash() {
```

Comment:
Can we rename this? It's not a hash, but a unique value. Maybe `generateUniqueValue`. Could we also either add to or replace this without the output of `Math.random()` to add more entropy? I'm always suspicious of timers as sources of uniqueness.

Reply:
Done.